test/src/unit-cbor.cpp

```diff
@@ -36,6 +36,7 @@ using nlohmann::json;
 #include <fstream>
 #include <sstream>
 #include <iomanip>
+#include <iostream>
```

Comment:
There should not be output in the unit tests.

Reply:
oof ouch, my plebian printf debugging has been revealed.